build/commands/lib/testUtils.js

```diff
@@ -28,16 +28,28 @@ const getChromiumTestsSuites = (config) => {
   return getTestsToRun(config, 'chromium_unit_tests').concat(['browser_tests'])
 }
 
-const getTestsToRun = (config, suite) => {
-  const testDepFile = path.join(config.outputDir, `${suite}.json`)
+const getTestGroupDeps = (filename) => {
+  const testDepFile = path.join(config.outputDir, filename)
   if (fs.existsSync(testDepFile)) {
-    suiteDepNames = JSON.parse(
+    let suiteDepNames = JSON.parse(
       fs.readFileSync(testDepFile, { encoding: 'utf-8' }),
     )
     return suiteDepNames.map((x) => x.split(':').at(-1))
+  } else {
+    return []
+  }
+}
+
+const getTestsToRun = (config, suite) => {
+  let testsToRun =
+    getTestGroupDeps(path.join(config.outputDir, `${suite}.json`))
+
+  if (config.isCI && config.targetOS == "linux") {
```

Comment:
This is to replicate the behavior we had before https://github.com/brave/brave-core/pull/30271